src/framework/mpas_io.F

```diff
@@ -5033,6 +5037,149 @@ subroutine MPAS_io_get_att_real1d(handle, attName, attValue, fieldname, precisio
 
    end subroutine MPAS_io_get_att_real1d
 
+   function handle_put_att_pio_redef(handle) result (pio_ierr)
+      implicit none
+      type(MPAS_IO_Handle_type), intent(inout) :: handle
+      integer :: pio_ierr
+
+      call mpas_log_write('Calling PIO_redef')
+      pio_ierr = PIO_redef(handle % pio_file)
+      if (pio_ierr /= PIO_noerr) then
+         io_global_err = pio_ierr
+         return
```

Comment:
I don't think this return statement is needed (except to avoid printing out the log message below).